app/views/pages/donations.erb

```diff
@@ -45,21 +45,21 @@
           <td><%= link_to('supertux@nerdpol.ch', 'https://nerdpol.ch/u/supertux') %></td>
           <td><%= link_to('SuperTux88', 'https://github.com/SuperTux88') %></td>
           <td><%= link_to('SuperTux88', 'https://liberapay.com/SuperTux88') %></td>
-          <td><%= t '.supertux88_work' %></td>
+          <td><%= t '.development' %>: <%= t '.federation_role' %>, Back-End, <%= t '.reviews_role' %>, <%= t '.release_management_role' %></td>
         </tr>
         <tr>
           <td>Fla</td>
           <td><%= link_to('fla@diaspora-fr.org', 'https://diaspora-fr.org/u/fla') %></td>
           <td><%= link_to('flaburgan', 'https://github.com/Flaburgan') %></td>
           <td><%= link_to('fla', 'https://liberapay.com/fla') %></td>
-          <td><%= t '.fla_work' %></td>
+          <td><%= t '.development' %>: Front-End, <%= t '.testing_role' %>. <%= t '.podmin_role' %> diaspora-fr.org, <%= t '.outreach_role' %></td>
```

Comment:
Ditto 'Front-end' - translatable?

Also, 'end' shouldn't be capitalised in either instance. Back-end and Front-end, not Back-End and Front-End.